### PR TITLE
ragg2: -f support for Python and Javascript

### PR DESCRIPTION
**Pull request by guedou**
This PR attempts to provide a fix for issue #8423. I am not happy with the Makefile kung-fu, it might be better to move r_print_code out of libr_core.

I also modified the way C is printed, however it changes the output.

### CODE REVIEW

**Comment by Maijin** [comment]
Dup Pull request https://github.com/radare/radare2/pull/8444

File: binr/ragg2/ragg2.c [hunk 1]
```diff
@@ -468,17 +469,13 @@ int main(int argc, char **argv) {
 				eprintf ("No format specified wtf\n");
 				goto fail;
 			}
+			RPrint *p = r_print_new();
```
Comment (XVilka) [inline binr/ragg2/ragg2.c]: Please put the space between function name and parentheses.

Comment (guedou) [reply binr/ragg2/ragg2.c]: Thanks for your review. It is now fixed and rebased.

**Comment by queenp** [comment]
Somehow seems to have broken ragg2?

"C:\projects\radare2-shvdd\build\radare2.sln" (default target) (1) ->
"C:\projects\radare2-shvdd\build\binr/ragg2\ragg2@exe.vcxproj.metaproj" (default target) (85) ->
"C:\projects\radare2-shvdd\build\binr\ragg2\ragg2@exe.vcxproj" (default target) (86) ->
(Link target) -> 
  ragg2.obj : error LNK2019: unresolved external symbol _r_print_code referenced in function _main [C:\projects\radare2-shvdd\build\binr\ragg2\ragg2@exe.vcxproj]
  .\ragg2.exe : fatal error LNK1120: 1 unresolved externals [C:\projects\radare2-shvdd\build\binr\ragg2\ragg2@exe.vcxproj]

**Comment by guedou** [comment]
@xarkes can you help me fixing this ?

**Comment by xarkes** [comment]
Yep, will check tomorrow.

File: binr/ragg2/Makefile [hunk 2]
```diff
@@ -3,6 +3,7 @@ BINDEPS=r_egg r_bin r_syscall r_asm
 BINDEPS+=r_io r_socket
 BINDEPS+=r_lang r_parse r_anal r_reg r_flag
 BINDEPS+=r_magic r_cons r_util r_search
+BINDEPS+=r_core r_hash r_debug r_fs r_config r_bp r_bp r_crypto
```
Comment (xarkes) [inline binr/ragg2/Makefile]: You also have to add the required dependencies in the `binr/ragg2/meson.build` file.
```
executable('ragg2', 'ragg2.c',
  include_directories: [platform_inc],
  link_with: [r_util, r_egg, r_bin],
  install: true
)
```
Obviously add it to the `link_with` line.

Comment (guedou) [reply binr/ragg2/Makefile]: Thanks a lot ! I pushed the modifications.

**Comment by xarkes** [comment]
Thanks!

**Pull request merged**